.github/workflows/abm_benchmark_test.yml

```diff
@@ -0,0 +1,30 @@
+# experimental pipeline to test benchmarks on github runners
+
+name: abm_benchmark_test
+
+on:
+  schedule:
+    - cron: "0 3,9,15,21 * * *" #every day on a few different hours
```

Comment:
Maybe change to 0,3,8,12,21? 
I guess this will clutter CI and 8 and 12 are probably where you don't need it in contrast to 9 and 15, but just my guess…

Reply:
The benchmark doesn't use much resources, so it doesn't really matter when exactly it runs as long as it covers different times throughout the day to see possible runtime variances